src/main/java/racingGame/business/CarFactory.java

```diff
@@ -3,21 +3,23 @@
 import java.util.ArrayList;
 import java.util.List;
 import racingGame.model.Car;
-import racingGame.model.Rules;
 
 public class CarFactory {
     
     private final List<Car> cars = new ArrayList<>();
-    private final Rules rules;
-    
-    public CarFactory(Rules rules) {
-        this.rules = rules;
-    }
-    
-    public List<Car> createCars() {
-        while(rules.isLimitCars(this.cars.size())) {
-            this.cars.add(new Car());
+
+    public List<Car> createCars(String[] names) {
```

Comment:
```suggestion
    public static List<Car> createCars(String[] names) {
```
클래스 메서드로 구현하는 것은 어떨까?